app/src/main/java/com/release/keyneez/presentation/main/explore/popular/PopularViewModel.kt

```diff
@@ -0,0 +1,51 @@
+package com.release.keyneez.presentation.main.explore.popular
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.release.keyneez.R
+import com.release.keyneez.domain.model.ExploreData
+
+class PopularViewModel : ViewModel() {
+    private val _itemList = MutableLiveData<List<ExploreData>>()
+    val itemList: LiveData<List<ExploreData>>
+        get() = _itemList
+
+    init {
+        getPopularItemList()
+    }
+
+    private fun getPopularItemList() {
```

Comment:
```suggestion
    private fun getPopularActivityList() {
```
Item 이라는 단어가 모호한 것 같습니다!